package.json

```diff
@@ -11,6 +11,6 @@
 		"prepare": "husky install"
 	},
 	"devDependencies": {
-		"husky": "7.0.4"
+		"husky": "^7.0.4"
```

Comment:
Why adding this? We prefer fixed versions... just 7.0.4... also for all other package.json - you can remove the ^